docs/apis/api_1.adoc

```diff
@@ -0,0 +1,390 @@
+= [big blue]*Alcor Controller APIs*
+:stylesheet: style1.css
+
+The followings are the RESTful APIs of Alcor controller system.
+
+[width="100%",cols="25%,15%,39%,21%"]
+|===
+|*Group* |*Method* |*Request*|*Response*
```

Comment:
The response column: 

What is Json(Vpc state)? The end user doesn't know about what the exact content a VPC state is. 

A better way is to provide a sample response.

Reply:
Hello Liguang,

I agree with yout opinion. I am working on it now.